### ISSUE TITLE
No connection on Wi-Fi

### ISSUE THREAD

**Issue by Teoshyu** [NekoX-Dev/NekoX#606]
**Describe the bug**
The program has no connection over Wi-Fi. Connects only on mobile data.
The problem also occurres on other versions of Nekogram X.
The problem does not occur on Telegram, Telegram FOSS, Nekogram.

**To Reproduce**
n/a

**Logcat**
attached

**Expected behavior**
n/a

**Screenshots**
attached

**Smartphone**
 - Device: H3113
 - OS: Android9
 - Version: Nekogram X v7.9.3 arm64-v8a miniAppleEmoji releaseNoGcm

**Additional context**
Thank you for looking into it.

[NekoX-1628520862491.log](https://github.com/NekoX-Dev/NekoX/files/6955451/NekoX-1628520862491.log)
![Screenshot_20210809-165631](https://user-images.githubusercontent.com/88306543/128729099-c115d658-ad95-4e27-8824-da89f3522cf7.png)

**Comment by coderkei** [comment]
Did you try reinstalling NekoX? I had the exact same problem and once I reinstalled it, it worked fine. I am not sure of any cause but I assume it had some kind of internal server list that may have become corrupt.

**Comment by Teoshyu** [comment]
Yes, perhaps. I did try to reinstall. Now, after the last update (8.0.0) it is working fine. So I guess this issue can now be closed.

Thanks for the reply.

**Comment by Teoshyu** [comment]
Unfortunately, the issue is still consistent as of the version 8.0.0 in my case.

It worked after the update for some time, now it doesn't.

Nekogram works fine, Nekogram X does not connect over Wi-Fi.

**Comment by Teoshyu** [comment]
Version: Nekogram X v8.0.0 arm64-v8a miniAppleEmoji releaseNoGcm

Tried both NekoX full and mini.
